scripts/run-draft-pr-scripts.ts

```diff
@@ -142,37 +142,12 @@ class DraftPRScriptRunner {
     
```

Comment:
the `constructor` in `run-draft-pr-scripts` should remove the `branchName` param and instead look it up from the currently checked out branch